src/migrateContentServer.test.ts

```diff
@@ -285,9 +339,14 @@ describe("migrateContentServer", () => {
   });
 
   it("keeps the 5.0 version of the widget, when migrating from 4.3 to 5.1 with `behaviorOnError` set to `keep-last-successful-version` and the 5.0 to 5.1 step fails", async () => {
+    const legacyUIFolderWithInvalidWidgets = addLegacyBookmarkToUIFolder(
```

Comment:
I would also find it nice to have a dedicated test resource (e.g. called `legacyUIFolderWithInvalidWidgets.ts`), exporting this. My rationale is that future us are more likely to notice that this already exists in case we need to use it in another test. We will be less likely to recreate the same test resource multiple times.

Reply:
Done, this got moved to `__test_resources__/legacyUIFolderWithInvalidWidgets.ts` ✅